design_wiki/seeds/Index.md

```
@@ -0,0 +1 @@
+This is the `seed` directory, for newly-formed notes and ideas.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)